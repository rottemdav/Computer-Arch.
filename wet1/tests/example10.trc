16 1 20 1 global_history local_tables not_using_share
0x5d4 T 0x2c2ff28
0xf2c N 0x69c7c
0xb90 T 0x7075edc
0x5e878 N 0xfa8
0x218 T 0x5e10
0xf4c T 0xca520
0xd9e374 N 0xf4db5b0
0xb60fb1c4 T 0x539744
0x94ff5ea0 T 0x8cd230
0x5e878 N 0xfa8
0x09fc1a0 N 0x5e81c
0xf8083624 T 0x88ae9f40
0x7d488 N 0x2c1e8
0xcdf41c T 0xd11a254
0x960 T 0x7624
0xc61fd0 T 0x9e18c
0x218 T 0x5e10
0xb90 N 0x7075edc
0x218 N 0x5e10
0xbc6ada8 N 0xf04f764
0x960 N 0x7624
0xb60fb1c4 T 0x539744
0x5d4 N 0x2c2ff28
0xcdf41c T 0xd11a254
0xff488 N 0xa1015c
0xe3c0 T 0xb04aad8
0x94ff5ea0 T 0x8cd230
0x94ff5ea0 T 0x8cd230
0xf2c N 0x69c7c
0x794 N 0xdcf4
0x09fc1a0 N 0x5e81c
0xc3158 N 0x98ba44
0x88616760 T 0xb00
0x7d488 T 0x2c1e8
0xbc6ada8 T 0xf04f764
0x88616760 T 0xb00
0xff488 N 0xa1015c
0x960 N 0x7624
0xcdf41c T 0xd11a254
0x340 N 0xd97ec
0x340 N 0xd97ec
0x72b64 N 0xfa47f8
0xcdf41c N 0xd11a254
0x578 T 0xad0
0x5d4 T 0x2c2ff28
0xf8083624 N 0x88ae9f40
0x94ff5ea0 T 0x8cd230
0xf8083624 N 0x88ae9f40
0x5d4 N 0x2c2ff28
0x578 T 0xad0
0x578 N 0xad0
0xb60fb1c4 T 0x539744
0xf2c T 0x69c7c
0x58fea44 N 0x8027000
0xc61fd0 T 0x9e18c
0xbfd34424 N 0x4ec
0xb60fb1c4 N 0x539744
0xcdf41c T 0xd11a254
0xf4c N 0xca520
0x5d4 T 0x2c2ff28
0x5e878 T 0xfa8
0xc3158 N 0x98ba44
0x960 N 0x7624
0xbc6ada8 N 0xf04f764
0x88616760 N 0xb00
0xf4c N 0xca520
0x5d4 T 0x2c2ff28
0xb90 T 0x7075edc
0xb90 T 0x7075edc
0xa5decc N 0xc76fd0
0x960 T 0x7624
0xbfd34424 N 0x4ec
0xcdf41c N 0xd11a254
0x09fc1a0 T 0x5e81c
0x58fea44 N 0x8027000
0x92f4 T 0x9a47578
0x834 N 0xc957ce8
0xb60fb1c4 T 0x539744
0x5d4 N 0x2c2ff28
0xa5decc N 0xc76fd0
0x5e878 T 0xfa8
0x960 T 0x7624
0xa5decc T 0xc76fd0
0x834 N 0xc957ce8
0xe9074 T 0xf030
0x58fea44 T 0x8027000
0x5d4 T 0x2c2ff28
0xc61fd0 N 0x9e18c
0x7d488 N 0x2c1e8
0x960 T 0x7624
0xff488 N 0xa1015c
0xa5decc T 0xc76fd0
0xbfd34424 N 0x4ec
0xbc6ada8 N 0xf04f764
0x94ff5ea0 N 0x8cd230
0x5d4 N 0x2c2ff28
0x218 N 0x5e10
0xf2c N 0x69c7c
0xb60fb1c4 N 0x539744
0x92f4 T 0x9a47578
0x94ff5ea0 N 0x8cd230
0xbfd34424 T 0x4ec
0xe3c0 N 0xb04aad8
0x94ff5ea0 T 0x8cd230
0x5d4 T 0x2c2ff28
0x834 N 0xc957ce8
0x794 T 0xdcf4
0x794 N 0xdcf4
0x88616760 T 0xb00
0xb90 N 0x7075edc
0xe3c0 N 0xb04aad8
0xe9074 N 0xf030
0xa5decc T 0xc76fd0
0xe3c0 N 0xb04aad8
0x834 T 0xc957ce8
0x767b4 T 0x36b8bc
0x578 N 0xad0
0x88616760 T 0xb00
0x58fea44 T 0x8027000
0x92f4 N 0x9a47578
0x578 T 0xad0
0xe9074 T 0xf030
0xf2c T 0x69c7c
0xcdf41c T 0xd11a254
0x578 N 0xad0
0xbc6ada8 T 0xf04f764
0xe9074 T 0xf030
0xc61fd0 T 0x9e18c
0x72b64 T 0xfa47f8
0xc3158 T 0x98ba44
0xa86c T 0x99df0
0x94ff5ea0 T 0x8cd230
0x7d488 N 0x2c1e8
0x92f4 N 0x9a47578
0xb90 N 0x7075edc
0xb90 T 0x7075edc
0xf8083624 T 0x88ae9f40
0xff488 N 0xa1015c
0xff488 T 0xa1015c
0xe9074 T 0xf030
0x5d4 T 0x2c2ff28
0x72b64 T 0xfa47f8
0xc61fd0 N 0x9e18c
0x767b4 T 0x36b8bc
0x794 T 0xdcf4
0xcdf41c T 0xd11a254
0xb90 T 0x7075edc
0x5e878 T 0xfa8
0x936e96bc T 0xaa460
0xf8083624 T 0x88ae9f40
0xbfd34424 T 0x4ec
0x340 T 0xd97ec
0xbfd34424 T 0x4ec
0x767b4 N 0x36b8bc
0xbc6ada8 T 0xf04f764
0x340 T 0xd97ec
0xc61fd0 N 0x9e18c
0xf4c T 0xca520
0xff488 N 0xa1015c
0xa86c N 0x99df0
0x58fea44 T 0x8027000
0x09fc1a0 T 0x5e81c
0x794 T 0xdcf4
0x5d4 N 0x2c2ff28
0x936e96bc T 0xaa460
0x936e96bc N 0xaa460
0xa86c N 0x99df0
0xa5decc N 0xc76fd0
0xbc6ada8 N 0xf04f764
0x5d4 T 0x2c2ff28
0xf2c T 0x69c7c
0xc61fd0 N 0x9e18c
0x88616760 T 0xb00
0xb90 N 0x7075edc
0xd9e374 T 0xf4db5b0
0x09fc1a0 T 0x5e81c
0xb90 N 0x7075edc
0x5d4 N 0x2c2ff28
0xb90 T 0x7075edc
0xd9e374 T 0xf4db5b0
0xff488 T 0xa1015c
0xf4c T 0xca520
0xe9074 N 0xf030
0xf4c N 0xca520
0x72b64 T 0xfa47f8
0xd9e374 N 0xf4db5b0
0x960 N 0x7624
0xb90 N 0x7075edc