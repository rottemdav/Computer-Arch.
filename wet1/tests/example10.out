0x5d4 N 0x5d8
0xf2c N 0xf30
0xb90 N 0xb94
0x5e878 N 0x5e87c
0x218 N 0x21c
0xf4c N 0xf50
0xd9e374 N 0xd9e378
0xb60fb1c4 N 0xb60fb1c8
0x94ff5ea0 N 0x94ff5ea4
0x5e878 N 0x5e87c
0x9fc1a0 N 0x9fc1a4
0xf8083624 N 0xf8083628
0x7d488 N 0x7d48c
0xcdf41c N 0xcdf420
0x960 N 0x964
0xc61fd0 N 0xc61fd4
0x218 N 0x21c
0xb90 N 0xb94
0x218 T 0x5e10
0xbc6ada8 N 0xbc6adac
0x960 N 0x964
0xb60fb1c4 T 0x539744
0x5d4 N 0x5d8
0xcdf41c T 0xd11a254
0xff488 N 0xff48c
0xe3c0 N 0xe3c4
0x94ff5ea0 N 0x94ff5ea4
0x94ff5ea0 T 0x8cd230
0xf2c N 0xf30
0x794 N 0x798
0x9fc1a0 N 0x9fc1a4
0xc3158 N 0xc315c
0x88616760 N 0x88616764
0x7d488 N 0x7d48c
0xbc6ada8 N 0xbc6adac
0x88616760 N 0x88616764
0xff488 N 0xff48c
0x960 N 0x964
0xcdf41c T 0xd11a254
0x340 N 0x344
0x340 N 0x344
0x72b64 N 0x72b68
0xcdf41c T 0xd11a254
0x578 N 0x57c
0x5d4 N 0x5d8
0xf8083624 N 0xf8083628
0x94ff5ea0 N 0x94ff5ea4
0xf8083624 N 0xf8083628
0x5d4 N 0x5d8
0x578 T 0xad0
0x578 N 0x57c
0xb60fb1c4 T 0x539744
0xf2c N 0xf30
0x58fea44 N 0x58fea48
0xc61fd0 N 0xc61fd4
0xbfd34424 N 0xbfd34428
0xb60fb1c4 N 0xb60fb1c8
0xcdf41c T 0xd11a254
0xf4c T 0xca520
0x5d4 N 0x5d8
0x5e878 N 0x5e87c
0xc3158 N 0xc315c
0x960 N 0x964
0xbc6ada8 N 0xbc6adac
0x88616760 N 0x88616764
0xf4c N 0xf50
0x5d4 N 0x5d8
0xb90 N 0xb94
0xb90 T 0x7075edc
0xa5decc N 0xa5ded0
0x960 N 0x964
0xbfd34424 N 0xbfd34428
0xcdf41c T 0xd11a254
0x9fc1a0 N 0x9fc1a4
0x58fea44 N 0x58fea48
0x92f4 N 0x92f8
0x834 N 0x838
0xb60fb1c4 N 0xb60fb1c8
0x5d4 T 0x2c2ff28
0xa5decc N 0xa5ded0
0x5e878 N 0x5e87c
0x960 N 0x964
0xa5decc N 0xa5ded0
0x834 N 0x838
0xe9074 N 0xe9078
0x58fea44 N 0x58fea48
0x5d4 N 0x5d8
0xc61fd0 N 0xc61fd4
0x7d488 N 0x7d48c
0x960 N 0x964
0xff488 N 0xff48c
0xa5decc N 0xa5ded0
0xbfd34424 N 0xbfd34428
0xbc6ada8 N 0xbc6adac
0x94ff5ea0 N 0x94ff5ea4
0x5d4 T 0x2c2ff28
0x218 N 0x21c
0xf2c N 0xf30
0xb60fb1c4 N 0xb60fb1c8
0x92f4 N 0x92f8
0x94ff5ea0 N 0x94ff5ea4
0xbfd34424 N 0xbfd34428
0xe3c0 N 0xe3c4
0x94ff5ea0 N 0x94ff5ea4
0x5d4 T 0x2c2ff28
0x834 N 0x838
0x794 N 0x798
0x794 N 0x798
0x88616760 N 0x88616764
0xb90 N 0xb94
0xe3c0 N 0xe3c4
0xe9074 N 0xe9078
0xa5decc N 0xa5ded0
0xe3c0 N 0xe3c4
0x834 N 0x838
0x767b4 N 0x767b8
0x578 N 0x57c
0x88616760 T 0xb00
0x58fea44 N 0x58fea48
0x92f4 N 0x92f8
0x578 N 0x57c
0xe9074 N 0xe9078
0xf2c N 0xf30
0xcdf41c N 0xcdf420
0x578 N 0x57c
0xbc6ada8 N 0xbc6adac
0xe9074 T 0xf030
0xc61fd0 N 0xc61fd4
0x72b64 N 0x72b68
0xc3158 N 0xc315c
0xa86c N 0xa870
0x94ff5ea0 N 0x94ff5ea4
0x7d488 N 0x7d48c
0x92f4 N 0x92f8
0xb90 N 0xb94
0xb90 N 0xb94
0xf8083624 N 0xf8083628
0xff488 N 0xff48c
0xff488 N 0xff48c
0xe9074 N 0xe9078
0x5d4 N 0x5d8
0x72b64 N 0x72b68
0xc61fd0 N 0xc61fd4
0x767b4 N 0x767b8
0x794 N 0x798
0xcdf41c T 0xd11a254
0xb90 N 0xb94
0x5e878 N 0x5e87c
0x936e96bc N 0x936e96c0
0xf8083624 N 0xf8083628
0xbfd34424 N 0xbfd34428
0x340 N 0x344
0xbfd34424 T 0x4ec
0x767b4 N 0x767b8
0xbc6ada8 N 0xbc6adac
0x340 T 0xd97ec
0xc61fd0 N 0xc61fd4
0xf4c N 0xf50
0xff488 N 0xff48c
0xa86c N 0xa870
0x58fea44 N 0x58fea48
0x9fc1a0 N 0x9fc1a4
0x794 T 0xdcf4
0x5d4 N 0x5d8
0x936e96bc N 0x936e96c0
0x936e96bc T 0xaa460
0xa86c N 0xa870
0xa5decc N 0xa5ded0
0xbc6ada8 T 0xf04f764
0x5d4 N 0x5d8
0xf2c N 0xf30
0xc61fd0 N 0xc61fd4
0x88616760 N 0x88616764
0xb90 N 0xb94
0xd9e374 N 0xd9e378
0x9fc1a0 N 0x9fc1a4
0xb90 N 0xb94
0x5d4 T 0x2c2ff28
0xb90 N 0xb94
0xd9e374 N 0xd9e378
0xff488 N 0xff48c
0xf4c N 0xf50
0xe9074 N 0xe9078
0xf4c N 0xf50
0x72b64 N 0x72b68
0xd9e374 N 0xd9e378
0x960 N 0x964
0xb90 T 0x7075edc
flush_num: 96, br_num: 188, size: 881b
